storage/mdadm/pkg.yaml

```diff
@@ -4,22 +4,19 @@ shell: /bin/bash
 dependencies:
   - stage: base
   - image: "{{ .BUILD_ARG_PKGS_PREFIX }}/systemd-udevd:{{ .BUILD_ARG_PKGS }}"
-  - image: "{{ .BUILD_ARG_PKGS_PREFIX }}/util-linux:{{ .BUILD_ARG_PKGS }}"
 steps:
   - sources:
-      - url: http://www.kernel.org/pub/linux/utils/raid/mdadm/mdadm-{{ .MDADM_VERSION }}.tar.gz
-        destination: mdadm.tar.gz
-        sha256: 61a1c22477555364dea1862df7c3b4e8b9a53ee733bad320a45fda27dd0ed44f
-        sha512: a77af2b371ed27b7bd89b7991302282f0fc77999eb93d7959446fc4b1c28388582d08ff0abfc470a9f9a8c8036acb0158edc44d9a8d406f807ca28111bf74459
+      - url: http://deb.debian.org/debian/pool/main/m/mdadm/mdadm_{{ .MDADM_VERSION }}.orig.tar.xz
```

Comment:
HTTPS works here. kernel.org mirror gives 404

Reply:
actually might be a false tag or the build has not uploaded the tarballs, have seen it happen at times, let's hold on

Reply:
Debian (->Ubuntu), Arch and more bumped it: https://repology.org/project/mdadm/versions

Reply:
might be tarballs haven't been uploaded yet https://www.kernel.org/pub/linux/utils/raid/mdadm/

Reply:
Well, yes. But Debian seems to be quite a solid source so perhaps that's okay. Or should I revert the change and wait?

Reply:
Debian I believe removes sources if they upgrade and they no longer need it in any way.

Reply:
reverted to 4.3, 4.4 still hasn't appeared at kernel.org